lib/Promise.php

```diff
@@ -21,7 +23,7 @@ interface Promise
      * Note: You shouldn't implement this interface yourself. Instead, provide a method that returns a promise for the
      * operation you're implementing. Objects other than pure placeholders implementing it are a very bad idea.
      *
-     * @param callable(\Throwable|null $reason, $value) $onResolved `$reason` shall be `null` on
+     * @param callable(\Throwable|null $reason, TValue|null $value) $onResolved `$reason` shall be `null` on
```

Comment:
Accoding to https://psalm.dev/, there needs to be a space behind `callable`.

Reply:
I don't think so. https://psalm.dev/docs/annotating_code/type_syntax/callable_types/

Reply:
In that case it's probably due to `$reason` and `$value` which need to be removed?

Reply:
Hmm... not sure how to rewrite the message then: "`$reason` shall be `null` on success, `$value` shall be `null` on failure."

Reply:
But I think that PHPStan at least accepts the syntax with names. I don't know about Psalm though since I'm not using it.